docs/en_US/Compression/pruning.rst

```diff
@@ -0,0 +1,26 @@
+#################
+Pruning
+#################
+
+A common technique to increase sparsity in neural network model weights and activations is pruning.
```

Comment:
Suggests changing to a simple sentence，`increase sparsity` sounds a little bit strange? how about:
Pruning is a common technique for compressing models.

Reply:
Thanks for good suggestion, updated.